kevpy/SendIT
# #161773626 login as a user and recieve an auth token

#### What does this PR do?
Creates an endpoint to login as a user

#### Description of Task to be completed?
Have the following endpoint working
/api/v1/login `POST`

#### How should this be manually tested?
Clone this repo
Set up a Flask environment by running pip install on requirements.txt on a virtual environment.
Run flask run
Post jason data to the endpoint.
example data:
```
{
    "email": "test@email.com",
    "password": "password",
}
```
One should get an auth token successful login. 

#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
#161773626
